### PR TITLE
Syntax error resolved

### PR DESCRIPTION
**Pull request by musahaidari**
The `{}` should be `[]`, but since there is no binding we do not get error. In case we add one, then the syntax will be incorrect.

### CODE REVIEW

File: src/plugins/sqlite.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ declare var sqlitePlugin;
  * })
  *   .then((db: SQLite) => {
  *
- *     db.executeSql('create table danceMoves(name VARCHAR(32))', {}).then(() => {}).catch(() => {});
+ *     db.executeSql('create table danceMoves(name VARCHAR(32))', []).then(() => []).catch(() => {});
```
Comment (ihadeed) [inline src/plugins/sqlite.ts]: `.then(() => [])` should be `.then(() => {})`

Comment (ihadeed) [inline src/plugins/sqlite.ts]: `.then(() => [])` should be `.then(() => {})`

Comment (musahaidari) [inline src/plugins/sqlite.ts]: @I hameed.  Yes. You are right. Thank you for the point

**Comment by musahaidari** [comment]
@Ihadeed Applied the change.

**Pull request merged**